vendor/composer/autoload_classmap.php

```diff
@@ -6,61 +6,27 @@
 $baseDir = dirname($vendorDir);
 
```

Comment:
Getting closer :)
 Appears you did not remove the phing stuff. See here for instructions on removing the phing stuff(step 4 and 5):
http://www.open-emr.org/wiki/index.php/Composer#Migrating